store/CSDK/main.cpp

```diff
@@ -55,6 +57,56 @@ JNIEnv *initJVM() {
     return env;
 }
 
+/**
+ * test read Schema from Index File
+ *
+ * @param env jni env
+ * @return whether it is success
+ */
+bool readSchemaInIndexFile(JNIEnv *env) {
+    printf("\nread Schema from Index File:\n");
+    CarbonSchemaReader carbonSchemaReader(env);
+    jobject schema = carbonSchemaReader.readSchemaInIndexFile(
+            "../resources/carbondata/510199997055746_batchno0-0-null-510199277323454.carbonindex");
```

Comment:
should not have binary in the repo. (index file)

Reply:
no in repo